recipes/aaf/all/conanfile.py

```diff
@@ -0,0 +1,76 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+
+
+class AafConan(ConanFile):
+    name = "aaf"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://sourceforge.net/projects/aaf/"
+    description = "A  cross-platform SDK for AAF. AAF is a metadata management system and file format for use in professional multimedia creation and authoring."
+    topics = ("aaf", "multimedia", "crossplatform")
+    license = "AAF SDK PUBLIC SOURCE LICENSE"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+            "shared": [True, False],
+            "fPIC": [True, False]
+            }
+    default_options = {
+            "shared": False,
+            "fPIC": True
+            }
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def validate(self):
+        if self.settings.compiler == "Visual Studio" and self.settings.compiler.runtime in ("MT", "MTd"):
+            raise ConanInvalidConfiguration("Static runtime not supported")
+        if self.settings.os == "Linux":
+            raise ConanInvalidConfiguration("Linux not supported yet")
```

Comment:
Not supported by the recipe, or not supported by aaf?
Adding a FIXME would be nice.
```suggestion
        if self.settings.os in ("FreeBSD", "Linux"):
            raise ConanInvalidConfiguration("Linux is not (yet) supported by this recipe")
```

Reply:
AAF is supported as stated in the readme: https://sourceforge.net/p/aaf/code2/ci/master/tree/README.TXT#l26

I didn't manage to make it work yet and since I don't use Linux I have little time to dedicated to Linux/FreeBSD support.